ec2/ec2-create-instance.sh

```diff
@@ -221,8 +221,12 @@ sudo yum -q -y install ansible git nano
 git clone -b $DA_BRANCH https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible.git dataverse
 export ANSIBLE_ROLES_PATH=.
 ansible-playbook $VERBOSE_ARG -i dataverse/inventory dataverse/dataverse.pb --connection=local $GVARG
+touch /tmp/ansible_complete
 EOF
 
+# did AWS go AWOL?
+rsync -av -e "ssh -i $PEM_FILE" centos@$PUBLIC_DNS:/tmp/dataverse/ansible_complete /tmp/ && /bin/rm /tmp/ansible_complete
```

Comment:
Is the idea that if "ansible_complete" is missing that this rsync command will fail, which will result in the build being marked as failed?

Reply:
yes.